modules/storage-persistent-storage-aws-maximum-volumes.adoc

```diff
@@ -7,13 +7,17 @@
 
 By default, {product-title} supports a maximum of 39 EBS volumes attached to one
 node. This limit is consistent with the
-link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS
-volume limits].
+link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS volume limits].
+
+[IMPORTANT]
+====
+As a cluster administrator, you must use either in-tree volumes or Container Storage Interface (CSI) volumes, but never both at the same time. The maximum attached EBS volume number is counted separately for in-tree and CSI volumes.
+====
```

Comment:
For this part, I have a question, what does cluster-admin can only use one of the drivers at the same time mean?
For 4.6, CSO created 2 storageclasses by default, one for in-tree plugin, one for CSI driver.  Cluster-admin needs to do some configuration to remove one of storageclass? Or we just need to add a note to notify the end-user?

Reply:
It's up to the cluster admin to decide how they're going to use the storage classes. For example, they can mark one class as a default and ignore the other.

Reply:
I clarified that they must use either in-tree or CSI volumes "and respective storage classes." In the note, I wanted to avoid going into much detail about possible configs. Does this look good to you, @jsafrane?